### PR TITLE
Correct Link, Add EVD details and customer whitelisting URLs

### PR DESCRIPTION
**Pull request by shivani61**
-Corrected link in SDK-Usage section for configuration.host
-Add details on how to do IMEI provisioning for EVD
-Added URLs for customers to whitelist
-Changed link to Esper dev trial in SDK
-Updating docs for enterprise v1 and token info apis 
-Changed personal names to Tyrell Smith as an example name

### CODE REVIEW

File: docs/home/espercli.md [hunk 1]
```diff
@@ -133,19 +165,18 @@ $ espercli enterprise show [OPTIONS]
...
+Address          #123, Industrial Layout, Random Avenue
+Location         Santa Clara, CA
+Zip Code         12345
+Email            contact@foo.io
+Contact Person   Shiv Sundar
+Contact Number   +145678901234
 
 $ espercli enterprise show -j
-{"id": "595a6107-b137-448d-b217-e20cc58ee84d", "name": "Foo Enterprise", "display_name": "Foo", "registered_name": "Foo Enterprise", "address": "#123, Industrial Layout, Random Avenue", "location": "Santa Clara, CA", "zipcode": "12345", "email": "contact@foo.io", "contact_person": "Muneer", "contact_number": "+141234501234"}%
+{"Enterprise Id": "595a6107-b137-448d-b217-e20cc58ee84d", "Name": "Foo Enterprise", "Registered Name": "Foo Enterprise", "Address": "#123, Industrial Layout, Random Avenue", "Location": "Santa Clara, CA", "Zip Code": "12345", "Email": "contact@foo.io", "Contact Person": "Shiv Sundar", "Contact Number": "+145678901234"}%
```
Comment (kirananto) [inline docs/home/espercli.md]: Should we have personal details here... ?
@kzotter was suggesting we use Generic info in the docs right ?

File: docs/home/espercli.md [hunk 2]
```diff
@@ -172,16 +203,15 @@ $ espercli enterprise update [OPTIONS]
...
-contact_number   +145678901234
+Enterprise Id    595a6107-b137-448d-b217-e20cc58ee84d
+Name             Foo Enterprise
+Registered Name  Foo Enterprise
+Address          #123, Industrial Layout, Random Avenue
+Location         Santa Clara, CA
+Zip Code         12345
+Email            contact@foo.io
+Contact Person   Muneer M
+Contact Number   +145678901234
```
Comment (kirananto) [inline docs/home/espercli.md]: (nit) same

Comment (shivani61) [reply docs/home/espercli.md]: changed all personal names to Tyrell Smith

File: docs/home/espercli.md (hunk 1 above)
Comment (shivani61) [reply docs/home/espercli.md]: changed all personal names to Tyrell Smith

**Pull request merged**